CHANGELOG.md

```diff
@@ -6,4 +6,5 @@
 - Add enablePremium handling
 - Handle restart gracefully
 - DevOps artifacts were ignored
-- Support custom scripts
\ No newline at end of file
+- Support custom scripts
+- Add severity level "debug" to logging (see [1232](https://dev.azure.com/cc-ppi/General/_workitems/edit/1232))
```

Comment:
Can you please move this to the top? We have no real versioning here, but the order should be newest to oldest

Reply:
I moved this to the top